### DISCUSSION TITLE
halo部署

### DISCUSSION THREAD

**Discussion by HongJunShan** [1Panel-dev/1Panel#5888] [Q&A]
您好，刚从其他面板换成1Panel不太熟悉，halo是容器化的，创建网站使用一键部署是相当于反向代理到halo容器端口吗？那这里外部访问地址作用是什么？在网站已经部署ssl证书后halo外部访问地址填写https+域名就会再访问域名就会显示502 Bad Gateway打不开为什么？http是可以的  谢谢！

**Comment by wanghe-fit2cloud** [comment]
是的，反代的 halo 容器服务。
外部访问地址的具体意思可以查看一下 halo 官方文档（1Panel 是采用的 halo 官方提供的 docker-compose 进行安装）
